src/pages/todo/Todo.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import { ShowPortal } from "../../components/Components";
+import { useTodo } from "../../Context/todo-context";
+import { getTasks } from "./TodoFunc";
+import { FaPlusCircle } from "react-icons/fa";
+import { Modal, TodoCard } from "../Pages";
+import "./todo.css";
+function Todo() {
+	const [show, setShow] = useState(false);
+	const { tasks, setTasks } = useTodo();
+
+	const modalHandler = () => setShow((val) => !val);
+
+	useEffect(() => {
+		setTasks(getTasks());
+	}, []);
+	return (
+		<div>
+			<main className="todo-container p-2 my-64 mx-auto rounded max-w-screen-md">
+				<header className="p-1 mb-1 text-center">
+					<h1 className="h2 text-6xl py-4 font-semibold">Welcome back</h1>
+					<p className="text-2xl font-serif mb-4">
+						You have {tasks.length} task for today.
```

Comment:
Just a small suggestion here would be to consider edge cases. 

For eg, {`You have ${tasks.length === 0 ? "no" : tasks.length} ${tasks.length > 1 ? "tasks" : "task"} for today.`}

The outcome of this would cover each scenario by showing 'You have no task for today', 'You have 1 task for today' and 'You have 2 tasks for today.'

Reply:
@tejas-muthya37 thank you for the suggestions, will do the changes.